torch/distributions/transforms.py

```diff
@@ -357,7 +358,7 @@ def _inverse(self, y):
         return y.log() - (-y).log1p()
 
     def log_abs_det_jacobian(self, x, y):
-        return -(y.reciprocal() + (1 - y).reciprocal()).log()
+        return -F.softplus(-x) - F.softplus(x)
```

Comment:
Nice! I think this is the best way to handle the numerical stability issue.

Reply:
This change looks great, but just to confirm, this still doesn't handle the `TransformedDistribution` case you mentioned?

Reply:
This is so nice!!!!!!!!!!!!!!!!!!!! A great way to address the overflow/underflow while computing `y`.